src/posix.mak

```diff
@@ -397,7 +397,7 @@ BACK_SRC = \
 	$C/machobj.c $C/mscoffobj.c \
 	$C/xmm.h $C/obj.h $C/pdata.c $C/cv8.c $C/backconfig.c $C/divcoeff.d \
 	$C/varstats.c $C/varstats.h $C/dvec.d \
-	$C/md5.c $C/md5.h \
```

Comment:
You still need to change `BACK_OBJS` + the VisualD filters.
BTW if you want to, we can do globbing over this list, s.t. it doesn't permanently need to be updated.

Reply:
I fixed the vcproj filters.